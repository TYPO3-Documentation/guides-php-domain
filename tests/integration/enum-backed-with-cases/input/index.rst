============================
PHP Enum, backed, with cases
============================

..  php:enum:: RedSuit
    :type: string

    In playing cards, a suit is one of the categories into which the
    cards of a deck are divided.

    ..  php:case:: Hearts
        :value: 'H'

        Hearts is one of the four suits in playing cards.

    ..  php:case:: Diamonds
        :value: 'D'

        Diamonds is one of the four suits in playing cards.
