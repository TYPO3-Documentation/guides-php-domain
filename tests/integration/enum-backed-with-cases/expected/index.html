<!-- content start -->
    <div class="section" id="php-enum-backed-with-cases">
            <h1>PHP Enum, backed, with cases</h1>
            <dl class="php enum">
    <dt class="sig sig-object php" id="redsuit">
        <em class="property"><span class="pre">enum</span> </em>
        <span class="sig-name descname">
    <span class="pre fqn"title="RedSuit">
        RedSuit
    </span>
</span><span class="pre"> : string</span>  <a class="headerlink" href="#redsuit" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

    <p>In playing cards, a suit is one of the categories into which the
cards of a deck are divided.</p>
<dl class="php case">
    <dt class="sig sig-object php" id="redsuit-hearts">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Hearts</span></span> : <span class="pre">&#039;H&#039;</span>  <a class="headerlink" href="#redsuit-hearts" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
    <p>Hearts is one of the four suits in playing cards.</p>
</dd>
</dl><dl class="php case">
    <dt class="sig sig-object php" id="redsuit-diamonds">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Diamonds</span></span> : <span class="pre">&#039;D&#039;</span>  <a class="headerlink" href="#redsuit-diamonds" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
    <p>Diamonds is one of the four suits in playing cards.</p>
</dd>
</dl>
    </dd>
</dl>
    </div>
<!-- content end -->
